src/installer/pkg/projects/netcoreapp/src/localnetcoreapp.override.targets

```diff
@@ -124,11 +140,58 @@
   <Target Name="OverrideCrossgenToolPaths" DependsOnTargets="GetCorePackagePaths" AfterTargets="GetCrossgenToolPaths" BeforeTargets="CreateCrossGenImages">  
     <PropertyGroup Condition="'$(CoreCLROverridePath)' != ''">
       <_runtimeDirectory>$(CoreCLROverridePath)</_runtimeDirectory>
-      <_crossgenPath>$(CoreCLROverridePath)/crossgen$(ApplicationFileExtension)</_crossgenPath>
+      <_crossgenPath>$([MSBuild]::NormalizePath('$(CoreCLROverridePath)', '$(_crossHostArch)', 'crossgen$(ApplicationFileExtension)'))</_crossgenPath>
       <_CoreLibSharedDirectory>$(CoreCLROverridePath)</_CoreLibSharedDirectory>
-      <_jitPath>$(CoreClrOverridePath)/$(_crossHostArch)/$(LibraryFilePrefix)clrjit$(LibraryFileExtension)</_jitPath>
+      <_jitPath>$([MSBuild]::NormalizePath('$(CoreCLROverridePath)', '$(_crossHostArch)', '$(LibraryFilePrefix)clrjit$(LibraryFileExtension)'))</_jitPath>
     </PropertyGroup>
+  </Target>
+
+  <!-- !!!BEGIN PATCHING Sharedfx SDK target -->
+  <Target Name="GetFilesFromPackageResolve">
```

Comment:
What part of this target did you have to override/patch?

Reply:
See below:

```
      <!-- !!!BEGIN CHANGE Persist existing TargetPath. -->
      <RidSpecificFilesToPackage Condition="'%(RidSpecificFilesToPackage.TargetPath)' == ''">
      <!-- !!!END CHANGE -->
```

Although I guess I neglected mentioning the original: [`<RidSpecificFilesToPackage>`](https://github.com/dotnet/arcade/blob/d2d8d9f5c14e353acc7dfcf69c5fa5a8c368af40/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.dependency.targets#L20).

Reply:
The paths without the change should work as-is. Were they not working correctly?

Reply:
In fact, the paths should have been the same as what they're overridden to here.

Reply:
The cross-target paths get reset by that, and need to be preserved:

https://github.com/dotnet/runtime/blob/4e23cd3deabebf6c1f269379eeb01dbadc4be22e/src/installer/pkg/projects/netcoreapp/src/localnetcoreapp.override.targets#L47-L49

I think they used to work by being added after `GetFilesFromPackageResolve` rather than before.

Reply:
Good catch!

Reply:
Well, I did have the benefit of seeing the build fail before I did this. 😄 When the target paths are the same between cross and target archs, the zip gets duplicate entries which fails an unzip step later.